.github/workflows/Issue-Needs-Attention.yml

```diff
@@ -1,12 +1,12 @@
+# NOTE: This is a common file that is overwritten by realm/ci-actions sync service
+# and should only be modified in that repository.
+
 name: Issue Needs Attention
 # This workflow is triggered on issue comments.
 on:
   issue_comment:
     types: created
 
-permissions: {}
```

Comment:
@cmelchior hmm - what are these permission changes solving? 
Should we do that in general across repos?